js-miniapp-sdk/src/miniapp.ts

```diff
@@ -10,6 +12,13 @@ interface MiniAppFeatures {
 
   /** @returns The Promise of permission result of mini app from injected side. */
   requestLocationPermission(): Promise<string>;
+
+  /**
+   * @param info The shared data must match the property in [ShareInfoType].
+   * @returns The Promise of load ad response result from injected side.
+   * Promise is rejected if failed to load.
+   */
+  shareInfo(info: ShareInfoType): Promise<null | Error>;
```

Comment:
On another PR I had suggested including `Error` in the type, but looks like this isn't working correctly. My editor isn't able to detect the return type correctly. So I think maybe this should be just `Promise<string>`?

Reply:
I have changed into `Promise<undefined>`. What do you think?

We don't return anything onSuccess because the miniapp just emits the shared info object.
The error string can be handled in onError so I guess we don't need `Error` in type.

Reply:
Updating to `Promise<string>`.